addons/mrp/models/stock_move.py

```diff
@@ -395,6 +418,7 @@ def _get_backorder_move_vals(self):
         return {
             'state': 'confirmed',
             'reservation_date': self.reservation_date,
+            'manual_consumption': self.bom_line_id.manual_consumption or self.product_id.tracking != 'none',
```

Comment:
It's the same than copy=False on the field no?

Reply:
Nope, our computed function are only for order in draft state, and the backorder are always confirmed. We need to manually pass the value.